static/js/operation.js

```diff
@@ -135,10 +139,159 @@ function parseTable(text, data) {
   data.types = types;
   data.features = [];
   data.df = df;
+  return [deci, cats, feats];
+}
+
+/**
+ * Parse data as assembly.
+ * @function parseAssembly
+ * @param {String} text - multi-line string in tsv format
+ * @param {Object} data - data object
+ * @returns {Array.<Object, Object, Object>} - decimals, categories and features
+ * @see cacheData
+ * This function duplicates the function of cacheData due to consideration of
+ * big data processing.
+ */
+function parseAssembly(text, data) {
+  var lines = splitLines(text);
+  if (lines.length === 1) throw 'Error: there is only one line.';
+
+  var format = getFileFormat(text); // read file format of the inputted text
+  var types = ['id'];
+  var cols = ['id', 'length', 'gc', 'coverage']; // the information available in the contig titles
+  var df = [];
+
+  var id = 0;
+  var length = 0;
+  var gc = 0;
+  var coverage = 0;
+  var contig_length = 0;
+  var GC_count = 0;
+
+  for (var i = 0; i < lines.length; i++) {
+    var line = lines[i];
+    if (line.toUpperCase().match(/^[ATGCU]+$/g)) { // Checking if the current line is not a contig title
+      [GC_count, contig_length] = calculateGCAndLength(line, GC_count, contig_length);
+      gc = ((GC_count/contig_length).toFixed(2)).toString(); // idnetified last component of contig title
+    }
+    else {
+      if (i !== 0) {
+        if (length <= 1000) {
+          continue;
+        }
+        df.push([id, length, gc, coverage]) // append dataframe with contig title information
+        GC_count = 0;
+        contig_length = 0;
+      }
+      [id, length, coverage] = parseContigTitles(line, format);
+      coverage = (Number(coverage).toFixed(3)).toString();
```

Comment:
Similarly, can `coverage` be kept as a floating number?

PS: Perhaps you converted `gc` and `coverage` to strings for a reason and I overlooked it. Please let me know if that's the case!